proctor-consumer/src/main/java/com/indeed/proctor/consumer/logging/ExposureLogger.java

```diff
@@ -0,0 +1,5 @@
+package com.indeed.proctor.consumer.logging;
+
+public interface ExposureLogger {
+    void createExposureLogEntry();
```

Comment:
my concern is log entry is very Indeed internal. (we are discussing about moving the proctor back to internal. but as of now it is still opensource)

Reply:
just my opinion, it may be better to make it more general.
Log entry is very specific.

One example is Datadog RUM feature flag integration. So instead of createExposureLogEntry, maybe something like logExposure

Reply:
updated